build-scripts/doc-gen/parser.ts

```diff
@@ -197,6 +202,20 @@ function visitNode(
           `Class ${node.name.getText()} has both a ` +
           `doc link alias and a doc decorator.`);
     }
+
+  // To parse a flag, the flag has to be defined as a `var` or `const` and attach `@doc` comments.
+  } else if (parseFlag && ts.isVariableDeclaration(node)) {
```

Comment:
Using `parseFlag && ts.isVariableDeclaration(node) && ts.isVarConst(node)` to constrain the `var` to be a constant may be better, but I am not sure why `ts.isVarConst` is unavailable.